autogen/variables.tf

```diff
@@ -354,6 +354,14 @@ variable "pod_security_policy_config" {
   }]
 }
 
+variable "resource_usage_export_config" {
+  description = "Enables network egress metering for this cluster. If enabled, a daemonset will be created in the cluster to meter network egress traffic."
+  default     = [{
```

Comment:
Is there a reason this needs to be a list? Seems like it should be an object: https://www.terraform.io/docs/configuration/types.html#object-

Reply:
I actually just copied the pod_security_policy_config variable, as I reused the same pattern for that implementation. My bad for blindly copying code without understanding it. Should we change it, or maybe both, to pure objects instead?

Reply:
We should change this one to be an object.

The other one could be changed in a future PR, but it's potentially a breaking change so I'd rather keep it disconnected.

Reply:
Fixed in #238